XTS/WS-C/dev/src/com/arjuna/webservices11/wscoor/client/WSCOORClient.java

```diff
@@ -41,12 +42,27 @@
     private static synchronized ActivationService getActivationService()
     {
         if (activationService.get() == null) {
-            // activationService.set(new ActivationService(null, new QName("http://docs.oasis-open.org/ws-tx/wscoor/2006/06", "ActivationService")));
-            activationService.set(new ActivationService());
+            setActivationService();
```

Comment:
I find it harder to read as you have to track around a class. I tend to just extract to methods I can see being reused. Anyway, you are the implementor - I was checking why you did it that way is all (considering the previous method performed the function inline in the method.